go/worker/common/p2p/peermgmt.go

```diff
@@ -162,6 +222,36 @@ func (mgr *PeerManager) watchRegistryNodes(consensus consensus.Backend) {
 			if nodeEv.IsRegistration {
 				_ = mgr.UpdateNode(nodeEv.Node)
 			}
+		case <-ticker.C:
+			mgr.logger.Debug("ticker fired")
```

Comment:
This  log message is probably leftover from testing, doesn't seem that useful to keeping it.